src/main/resources/EDIFACT-Common/IBM_EDI_Format.xsd

```diff
@@ -1,63 +1,63 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <!--***********************************************************************
-    * IBM hereby grant permission, to those obtaining a copy of these materials, 
-    * to use, copy and modify the materials solely for use with DFDL processors. 
-    * THESE MATERIALS ARE PROVIDED AS IS WITH NO WARRANTIES OF ANY KIND. 
+    * IBM hereby grant permission, to those obtaining a copy of these materials,
```

Comment:
I'm wondering if there is value in removing the whitespace commit for this repo? For example, maybe we'll want to merge daffodil-dev into main, or diff the files on main to see what changed specifically for Daffodil. It would probably make viewing diffs or merging much easier if we minimized whitespace changes.

Reply:
I was planning on not squashing that commit and leaving it in the commit history so that the changes shouldn't be too bad to reason about.

Reply:
Even if it's not squashed, the files still have 1000+ lines of whitespace changes making them difficult to diff agains files in the main branch. For example, `git diff master --find-renames` will show the diffs between daffodil-dev of those files. If daffodil-dev has a bunch of whitespace changes, that diff would be much more difficult to make sense of.

Reply:
I tend to want to avoid the whitespace changes because the Daffodil version of this schema is effectively a "fork" of the original IBM-created EDIFACT schema. Enhancements to this schema coming from the IBM community are likely to be contributed to the main IBM branch. We, or somebody, would then need to map those forward into the Daffodil branch. 

There are two ways that could go. (1) isolate their changes and map forward to the daffodil-dev branch (2) isolate all our changes on the daffodil-dev branch and reapply them to a new copy of the main branch. 

Either way whitespace changes make this more difficult. 

So I suggest roll back the whitespace-only changes also.